### PR TITLE
Linux desktop file and debian package name

### PR DESCRIPTION
**Pull request by ChrisLauinger77**
I also added the svg icon
which could be deployed to /usr/share/icons
but I don't know what to edit to achieve this :(
This PR addresses #391 & #473

### CODE REVIEW

**Comment by ChrisLauinger77** [comment]
Found the place in build.yml meanwhile - svg will be copied there

File: .pkg/DEBIAN/control [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Package: psst
+Package: psst-gui
```
Comment (jacksongoode) [inline .pkg/DEBIAN/control:1]: Is there a reason to rename this package?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -126,6 +126,8 @@ jobs:
             mkdir -p "${{runner.workspace}}/pkg/usr/share/icons/hicolor/${LOGO_SIZE}x${LOGO_SIZE}/"
             cp "./psst-gui/assets/${LOGO}" "$_/psst.png"
           done
+          mkdir -p "${{runner.workspace}}/pkg/usr/share/icons/hicolor/scalable/apps/"
+          cp "./psst-gui/assets/psst.svg" $_
```
Comment (jacksongoode) [inline .github/workflows/build.yml:130]: Could you call the SVG `logo.svg` to match the other assets included?

**Review by jacksongoode — CHANGES_REQUESTED** [review]
Looks good for the most part

File: .pkg/DEBIAN/control (hunk 1 above)
Comment (ChrisLauinger77) [reply .pkg/DEBIAN/control:1]: Yea #391

Comment (Insprill) [reply .pkg/DEBIAN/control:1]: https://github.com/jpochyla/psst/issues/391

File: .github/workflows/build.yml (hunk 2 above)
Comment (ChrisLauinger77) [reply .github/workflows/build.yml]: Done

**Pull request merged**